src/App.scss

```diff
@@ -4,5 +4,23 @@ iframe {
 
 html {
   font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
-  font-size: 24px;
+  font-size: 36px;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  padding-top: 150px;
+  background-image: url('https://cdn.pixabay.com/photo/2017/08/30/01/05/milky-way-2695569_960_720.jpg');
+  background-size: cover;
+  text-align: center;
+}
+
+.App {
+  transition: transform 0.5s ease;
+  color:white;
+
+  &:hover {
+    transform:scale(1.2);
+  }
 }
```

Comment:
Each component should be in a separate folder with tsx and scss files related to this component